Library/Homebrew/formula.rb

```diff
@@ -449,7 +449,7 @@ def installed_prefixes
   # All of current installed kegs.
   # @private
   def installed_kegs
-    installed_prefixes.map { |dir| Keg.new(dir) }
+    installed_prefixes.map { |dir| Keg.new(dir) }.sort_by { |keg| keg.version }
```

Comment:
You can simplify this a little bit: Use `.sort_by(&:version)` instead of what you appended.

I think this should be fine. (There are currently no assumptions being made in what order `installed_kegs` returns its results, thus changing that shouldn't cause any issues.)